drivers/overlay/ov_network.go

```diff
@@ -109,6 +109,11 @@ func (d *driver) DeleteNetwork(nid string) error {
 		return fmt.Errorf("could not find network with id %s", nid)
 	}
 
+	// Make sure driver resources are initialized before proceeding
+	if err := d.configure(); err != nil {
```

Comment:
After the introduction of the datastore update, `config()` now only contains the vxlan idm initialization.
But I agree if in future we add more stuff in `configure()`, then it is better to have it executed at the beginning of this function.